src/formatter/Cli.hx

```diff
@@ -138,6 +147,40 @@ class Cli {
 		}
 	}
 
+	function runPipe() {
+		var content:Null<Bytes> = null;
+		try {
+			var config = Formatter.loadConfig(Sys.getCwd());
```

Comment:
Hm.. using the cwd to load the config doesn't seem correct. I assumed you'd still need to pass the file path via `--source` so the correct config file can be used?

Reply:
(and perhaps it should error if there's more than one path or the path doesn't point to a file?)

Reply:
I was thinking about adding an option (or using `-s`) to set a reference point for config detection, but I don't think that's really necessary. You are calling formatter from a command line to feed it data via a pipe, I don't think it's too much to ask to set your current folder to where a `hxformat.json` exists. When called from an IDE as an external program, I would expect that your working directory is already set to your workspace root.

However I am pondering over the idea to add another option to specify a `hxformat.json` file, that will override config detection logic. Such an option would work with both `-s` and `--stdin`.

Reply:
I guess you could `cd` to the correct directory.. but I would've just assumed that `--source` works, and if I'm reading this correctly that would silently fail right now. So I think there's two options:

- specifying `--source` together with `--stdin` should throw an error
- `--source` should be mandatory with `--stdin` and be used for config

>When called from an IDE as an external program, I would expect that your working directory is already set to your workspace root.

You can have multiple `hxformat.json` files per workspace.

Reply:
see #413